### ISSUE TITLE
Swagger: call INVALID_TYPE

### ISSUE THREAD

**Issue by honeycrypto** [semuxproject/semux-core#106]
Semux version: `1.5.0-vmrc01`

When I try to make a simple CALL to existing contract address via Swagger UI, I'm receiving this error: 
```
{
  "success": false,
  "message": "Transaction rejected by pending manager: INVALID_TYPE",
  "result": null
}
```

**Comment by honeycrypto** [comment]
And CREATE always returns Unauthorized and shows pop-up again, while all other requests work

**Comment by orogvany** [comment]
Ah.  The unauthorized is because you're putting a huge thing on the URL and your browser (or semux) is truncating.  Set them as post body params.

Another reason why supporting web3js is necessary

**Comment by orogvany** [comment]
We can probably change the swagger doc to denote these as post body params by default

**Comment by orogvany** [comment]
@honeycrypto I'm still unable to get a repro for INVALID_TYPE

Will still look into the unauthorized from swagger.

**Comment by honeycrypto** [comment]
I can gzip my testnet DB if this will help you to investigate why wallet at latest block doesn't see that VM was activated

**Comment by orogvany** [comment]
@honeycrypto - yes please zip it up for me.  I think I know what's going on.  But want to verify

**Comment by orogvany** [comment]
So here's the problem.  Once rc1 activated, non updated validators kept on forging.  So by the time they updated, the logic is just to scan last 2000 blocks, but since it was activated, validators had stopped signalling.  If the validators had hung on activation woulda been fine, but since no new vm calls were made immediately on activation,they kept going forwards, so checking last 2k missed the activation window

**Comment by orogvany** [comment]
fixed by #139
